identity/extension_credentials.go

```diff
@@ -35,6 +35,7 @@ func (r *SchemaExtensionCredentials) Run(_ jsonschema.ValidationContext, s schem
 			}
 		}
 
+		// TODO--lowercasing something (?jsonnet? - ?no idea?)
```

Comment:
I'm not sure, but shouldn't we update this path as well?

Reply:
Not 100% sure but here it is d04ce2edc